v1/faasd.yaml

```diff
@@ -0,0 +1,50 @@
+description: A portable FaaS framework
```

Comment:
We have the description `a lightweight & portable faas engine` on our GitHub repo.